index.html

```diff
@@ -2532,183 +2532,169 @@ <h1>Servient Implementation</h1>
+            </p>
+            <p>
+                The <a>Consumed Thing</a> abstraction represents a remotely hosted <a>Thing</a> for <a>Consumers</a> that needs to be accessed using a communication protocol.
+                <a>Consumed Things</a> are proxy objects or stubs.
+                The behavior implementation is restricted to reading their metadata and activating their <a>Interaction Affordances</a> as described in the corresponding <a>TD</a>.
+                <a>Consumed Things</a> can also represent system features such as local hardware or devices behind proprietary or legacy communication protocols.
+                In this case, the <a>WoT Runtime</a> must provide the necessary adaptation between system API and <a>Consumed Thing</a>.
+                Furthermore, it must provide corresponding <a>TDs</a> and make them available to the behavior implementation,
+                for instance, by extending a discovery mechanism.
+            </p>
+            <p>
+                When using the <a>WoT Scripting API</a>, <a>Exposed Thing</a> and <a>Consumed Thing</a> are JavaScript objects,
+                which can be created, operated on, and descroyed by application scripts.
+                However, access my be restricted through security configuration,
+                for instance, in multi-tenant <a>Servients</a>.
             </p>
         </section>
         <section>
-            <h2>Life Cycle Management</h2>
-            <p> Certain operations are needed to manage the lifecycle
-                of an exposed Thing such as retrieving and/or serving
-                a Thing Description, instantiating an <a>Exposed Thing</a> abstraction based
-                on the Thing Description, and terminating an
-                <a>Exposed Thing</a>. The details of such operations vary among
-                different kinds of devices. A WoT Servient Life Cycle
-                Management (LCM) system encapsulates those lifecycle
-                operations. It uses internal interfaces to realize the
-                lifecycle management of exposed Things. The LCM is a
-                system component that integrates with the <a>WoT Runtime</a>.
-                The <a>WoT Scripting API</a> provides one implementation of the
-                LCM functionality.
-             </p>
-             <p> For a <a>WoT Client</a> or <a>Servient</a>, the lifecycle management of 
-                 <a>Consumed Things</a> is realized in the same manner as that
-                 of <a>Exposed Things</a>.
-             </p>
+            <h2>Private Security Configuration</h2>
+            <p>
+                Private security metadata is also conceptually
+                managed by the <a>WoT Runtime</a> but is intentionally not made
+                directly accessible to the application. In fact, in the
+                most secure hardware implementations, such private
+                security data is stored in a separate, isolated memory
```

Comment:
memory -> entity ?

Reply:
I would leave it with memory, as it referres to a concrete implementation strategy (e.g., with secure hardware elements, which offer such memory).